inc/report.class.php

```diff
@@ -166,14 +166,13 @@ static function title() {
     * @since 0.84
    **/
    static function showDefaultReport() {
-      global $DB;
+      global $DB, $CFG_GLPI;
 
       // Title
       echo "<span class='big b'>GLPI ".Report::getTypeName(Session::getPluralNumber())."</span><br><br>";
 
       // 1. Get counts of itemtype
-      $items     = ['Computer', 'Monitor', 'NetworkEquipment', 'Peripheral', 'Phone',
-                         'Printer', 'Software'];
+      $items     = $CFG_GLPI["asset_types"];
```

Comment:
This will replace `Software` by `SoftwareLicense`. I do not know if it is acceptable or not in this report. I guess it would be more logical to have a report about licences.